src/extractDates.js

```diff
@@ -6,11 +6,18 @@ export default extractDates
 const parser = new chrono.Chrono
 parser.refiners.push(wholeMonth)
 
-function extractDates(str) {
+function extractDates(str, timezoneOffset) {
```

Comment:
would this make sense with a default? e.g.

```javascript
function extractDates(str, timezoneOffset = 0) {
```

and we could do away with the `if (timezoneOffset)` below? or do we want this to be optional?